templates/srv/www/sites/composer.json.j2

```diff
@@ -0,0 +1,2 @@
+{% set _composer_json = (roundcube__register_composer_json.stdout | from_json) %}
+{{ _composer_json | to_nice_json }}
```

Comment:
Actually I'm not fully sure 😉  No, of course at the moment not. But I wanted to have a way to influence the `composer.json` so that I can remove/add some packages according to our needs. E.g. add the LDAP driver. The idea was, to install as much as possible via package manager and only the remaining via composer. But I didn't test yet, if that works.